File: 1019.cpp
#include <iostream>

using namespace std;

int main(){
    int segundos, h, m, s;
    cin >> segundos;
    h = segundos / 3600;
    m = (segundos%3600) / 60;
    s = segundos % 60;
    cout << h << ":" << m << ":" << s << endl;
    return 0;
}

File: 1019.py
##s = segundos do input
s = int(input())
horas = s // 3600
s = s - horas * 3600
minutos = s // 60
s = s - minutos * 60
##segundos da resposta
segundos = s
print('{}:{}:{}'.format(horas, minutos, segundos))
